AdobeStockImageAdminUi/Test/Mftf/Test/UserSearchesStockImagesByKeywords.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="UserSearchesStockImagesByKeywords">
```

Comment:
The test contains many actions that may be reused within the scope of other tests. However, until they are hardcoded in the test, they cannot be reusable. The best practice is to make the test consist of action groups only. At the same time, the action groups should be granular (small) to match the "reusable" principle. 
The main idea is another test can pick up a set of existing action groups and use them in the test body. 
Please, check my comments for this test https://github.com/magento/adobe-stock-integration/pull/239/files#diff-f63ad1182db6cace7d59611da116115fR40 and recent MFTF implementations https://github.com/magento/magento-functional-tests-migration/pull/707/files for examples

Reply:
@rogyar All action moved to action group, in test I wrote only assertions.